packages/cli/src/commands/setup/gcp_firebase/gcp_firebase.js

```diff
@@ -0,0 +1,171 @@
+    })
+    .option('db-name', {
+      description: 'Name of Cloud SQL PostgreSQL database',
+      demand: true,
+      type: 'string',
+    })
+    .option('db-password', {
+      description: 'Password to use in Cloud SQL connection string',
+      demand: true,
+      type: 'string',
+    })
+    .option('region', {
+      description: 'Region Cloud Run container is running in',
+      default: 'us-central1',
+      type: 'string',
+    })
+    .option('api-routes', {
+      description: 'Routes that should be API cloud run container',
+      default: ['/graphql'],
+    })
+    .parserConfiguration({
+      'greedy-arrays': false,
+    })
+}
+
+export const handler = async ({
+  force,
+  apiRoutes,
+  containerName,
+  dbName,
+  dbPassword,
+  project,
+  region,
+  site,
+}) => {
+  const tasks = new Listr([
+    {
+      title: 'Configuring firebase project aliases...',
+      task: () => {
+        const template = lt(
+          fs.readFileSync(
+            path.resolve(__dirname, 'templates', 'firebaserc.template'),
+            'utf8'
+          )
+        )
+        return writeFile(
+          path.join(getPaths().base, '.firebaserc'),
+          template({ project, site }),
+          { overwriteExisting: force }
+        )
+      },
+    },
+    {
+      title: 'Configuring firebase project...',
+      task: () => {
+        const template = lt(
+          fs.readFileSync(
+            path.resolve(__dirname, 'templates', 'firebase.json.template'),
+            'utf8'
+          )
+        )
+        return writeFile(
+          path.join(getPaths().base, 'firebase.json'),
+          template({ apiRoutes, containerName, project, region, site }),
+          { overwriteExisting: force }
+        )
+      },
+    },
+    {
+      title: 'Generating cloudbuild.yaml for deployment',
+      task: () => {
+        const template = lt(
+          fs.readFileSync(
+            path.resolve(__dirname, 'templates', 'cloudbuild.yaml.template'),
+            'utf8'
+          )
+        )
+        return writeFile(
+          path.join(getPaths().base, 'cloudbuild.yaml'),
+          template({
+            apiRoutes,
+            containerName,
+            databaseSubstitution: '${_DATABASE_URL}',
+            dbName,
+            project,
+            region,
+            site,
+          }),
+          { overwriteExisting: force }
+        )
+      },
+    },
+    {
+      title: 'Generate .env file with DB connection string',
+      task: () => {
+        const template = lt(
+          fs.readFileSync(
+            path.resolve(__dirname, 'templates', 'env.template'),
+            'utf8'
+          )
+        )
+        return writeFile(
+          path.join(getPaths().base, '.env'),
+          template({
+            apiRoutes,
+            containerName,
+            dbPassword,
+            dbName,
+            project,
+            region,
+            site,
+          }),
+          { overwriteExisting: force }
+        )
```

Comment:
Maybe we should use write `appendFile` instead, just in case a user already has a `.env` file?

Reply:
Sorry, just saw your notes in the pull-request above about requiring more than 1 `.env` file... I'm doing some research.